kmp_client/grpcprotos/src/main/proto/com/dobby/vpnserver/vpnserver.proto

```
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package grpcproto;
```

Buf (1.65.0)

[error] Files with package "grpcproto" must be within a directory "grpcproto" relative to root but were in directory "kmp_client/grpcprotos/src/main/proto/com/dobby/vpnserver".

(PACKAGE_DIRECTORY_MATCH)